ffi/mupdf.lua

```diff
@@ -343,35 +337,37 @@ local function is_unicode_wspace(c)
         c == 0x3000 --  Ideographic space
 end
 local function is_unicode_bullet(c)
-    -- The last 2 aren't strictly bullets, but will do for our usage here
+    -- Not all of them are strictly bullets, but will do for our usage here
     return c == 0x2022 or --  Bullet
         c == 0x2023 or --  Triangular bullet
+        c == 0x25a0 or --  Black square
```

Comment:
Is there supposed to be an order to these?

Reply:
I don't know if there is any order here — maybe number of occurances? —, so the three I added has been simply ordered by value.